elements/rh-pagination/demo/no-numeric-control.html

```diff
@@ -0,0 +1,22 @@
+<rh-pagination>
```

Comment:
Resolve when ready to merge

Reply:
gonna pre-empt corey on this one, assuming he's cool with this change, as he brought it up originally

Reply:
@bennypowers Looks good, thanks!